.github/workflows/review.yml

```
@@ -33,9 +33,9 @@ jobs:
     needs: [scalingo-link]
     steps:
       - name: Add review app URL
-        uses: blablacar/action-sticky-description@df5650c838f2bbc860ab8edd059b6e255fd7bb82
+        uses: thollander/actions-comment-pull-request@v2
         with:
-          issue_number: ${{ github.event.number }}
-          marker: "## emplacement de l'URL de la review app ##"
+          pr_number: ${{ github.event.number }}
           message: |
             Se rendre sur la [review app](https://cdb-app-review-pr${{ github.event.number }}.osc-fr1.scalingo.io).
```

je propose de modifier le message du commentaire par 
```suggestion
            La review app a été déployée à [cette adresse](https://cdb-app-review-pr${{ github.event.number }}.osc-fr1.scalingo.io).
```